src/app/view/history.rs

```diff
@@ -73,6 +75,7 @@ impl HistoryView {
         warning: Option<&Error>,
         events: Vec<Element<'a, Message>>,
         event_kind_filter: &Option<HistoryEventKind>,
+        is_next: bool,
```

Comment:
nit: maybe `has_next`? It seems to me that this is used for checking if we have to add another page...